README.md

```diff
@@ -83,6 +83,33 @@ This can also be a string accepted by the
 
 Serve files relative to `path`.
 
+##### fs
+
+Use a filesystem other than the node-js built-in `fs` module. It must support the following methods:
+
+ * stat(path, callback)
+ * createReadStream(path, options)
+
+The resulting stat object from `stat` must (at least) look like:
```

Comment:
The types of `err` objects also need to be documented, for instance, we look for `ENOENT` in the `code` property and do special things, so people implementing a fake `stat` would need to know this.